cluster/gce/gci/README.md

```diff
@@ -1,11 +1,42 @@
-# Container-VM Image
+# Container-Optimized OS
 
-[Container-VM Image](https://cloud.google.com/compute/docs/containers/vm-image/)
-is a container-optimized OS image for the Google Cloud Platform (GCP). It is
-primarily for running Google services on GCP. Unlike the open preview version
-of container-vm, the new Container-VM Image is based on the open source
-ChromiumOS project, allowing us greater control over the build management,
+[Container-Optimized OS](https://cloud.google.com/container-optimized-os/docs),
+(previously Google Container-VM image a.k.a GCI) is a container-optimized OS image for the Google Cloud Platform (GCP). It is
+primarily for running Google services on GCP. Container-Optimized OS is an open
+source OS based on
+the open source [ChromiumOS project](https://www.chromium.org/chromium-os), allowing us greater control over the build management,
 security compliance, and customizations for GCP.
 
+Container-Optimized OS is open [source](https://cos.googlesource.com), and is released on milestones. Example milestones are
+81,  84, 85. Each milestone will experience three release channels -- dev, beta and stable to reach
+stability. The promotion between those channels are about six weeks.
+Starting milestone 69, for
+every 4 milestones, the last milestone will be promoted into LTS image.
+For details, please see COS's [Release Channels](https://cloud.google.com/container-optimized-os/docs/concepts/release-channels)and [Support
+Policy](https://cloud.google.com/container-optimized-os/docs/resources/support-policy).
+
+## COS in End-to-End tests
+
+Container-Optimized OS images are used by kubernetes [End-to-End tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md) and
+[Node End-to-End tests](https://github.com/kubernetes/community/tree/master/contributors/devel/sig-node). To see current
+active releases, please refer to COS's [Release
+Notes](https://cloud.google.com/container-optimized-os/docs/release-notes). When
+choose images for testing, the latest LTS images is suggested for running
+presubmit, postsubmit or periodics tests since the image is stable and has
+latest bug and security fixes. For new features which need new kernel features,
+the latest dev or beta or stable channel can be used. When multiple COS images
```

Comment:
Do we have any examples of dev/beta usage where we actually need it? I see some usages of dev-83 and beta-81, which seems wrongheaded at this current time of 81 being lts.

Reply:
I do not have context on beta-81/dev-83. However, if we need new features and continuous integration/testing, we need those images.